### PR TITLE
Week - 3 Assignments submission.

### PR DESCRIPTION
**Pull request by GopalDev22**
Hi Suraj,

 I completed all assignments. except one assignment related to Bootstrap Library from session-07. 

Please review and merge the code.

Thanks,
Venu Gopal.

### CODE REVIEW

**Comment by suraj7091** [comment]
assignment is not as described in doc

File: Week-3/submission/Session-09_JS_Basics/JavaScript arrays/script.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    } else {
+      evenNum.push(sortArr[i]);
+    }
+  }
+  console.log("Even Number:");
+  for (let i = 0; i < evenNum.length; i++) {
+    if (evenNum.length !== 0) {
+      console.log(evenNum[i]);
+    } else {
+      console.log("no even number");
```
Comment (suraj7091) [inline Week-3/submission/Session-09_JS_Basics/JavaScript arrays/script.js:18]: you should output none for both odd and even

File: Week-3/submission/Session-09_JS_Basics/JavaScript game object/script.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  newGame: function () {
+    let newValue = this;
+    if (this.lives) {
+      newValue.lives = 3;
+      newValue.coins = 0;
+      return console.log(game);
+    }
+  },
+};
+console.log(game);
```
Comment (suraj7091) [inline Week-3/submission/Session-09_JS_Basics/JavaScript game object/script.js]: image
your output is not matching , Your function calls should be rearranged

File: Week-3/submission/Session-09_JS_Basics/JavaScript game object/script.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  get points() {
+    let point = this;
+    point.coins = 2;
+
+    console.log(`points= ${this.coins * 10}`);
+  },
+  playerDies: function () {
+    let chance = this;
+    if (this.lives !== 0) {
+      chance.lives = this.lives - 1;
```
Comment (suraj7091) [inline Week-3/submission/Session-09_JS_Basics/JavaScript game object/script.js]: this.lives>0 , this should be the condition

**Review by suraj7091 — CHANGES_REQUESTED** [review]

File: Week-3/submission/Session-09_JS_Basics/JavaScript game object/script.js (hunk 2 above)
Comment (GopalDev22) [reply Week-3/submission/Session-09_JS_Basics/JavaScript game object/script.js]: Fixed the changes. please review.

File: Week-3/submission/Session-09_JS_Basics/JavaScript game object/script.js (hunk 3 above)
Comment (GopalDev22) [reply Week-3/submission/Session-09_JS_Basics/JavaScript game object/script.js]: Fixed the changes. please review.

File: Week-3/submission/Session-09_JS_Basics/JavaScript arrays/script.js (hunk 1 above)
Comment (GopalDev22) [reply Week-3/submission/Session-09_JS_Basics/JavaScript arrays/script.js:18]: Fixed the changes. please review.

**Review by suraj7091 — APPROVED** [review]

**Pull request merged**